packages/dashboard/src/components/inlineInputForm/stories/index.js

```diff
@@ -28,15 +22,30 @@ import InlineInputForm from '..';
 export default {
   title: 'Dashboard/Components/InlineInputForm',
   component: InlineInputForm,
+  args: {
+    error: 'error',
+  },
+  argTypes: {
+    onEditCancel: {
+      action: 'onEditCancel',
+    },
+    onEditComplete: {
+      action: 'onEditComplete',
+    },
+  },
+  parameters: {
+    controls: {
+      include: ['error', 'onEditCancel', 'onEditComplete'],
+    },
+  },
 };
 
-export const _default = () => (
+export const _default = (args) => (
   <InlineInputForm
-    onEditComplete={(newValue) => action('onEditComplete')(newValue)}
-    onEditCancel={action('onEditCancel')}
+    onEditComplete={(newValue) => args.onEditComplete(newValue)}
     value={'some input value'}
     id={'898989'}
     label="my hidden input label"
-    error={text('error', '')}
+    {...args}
   />
```

Comment:
💭 For the future: would be good to add some padding around this input so the focus styles are not cut off
![image](https://user-images.githubusercontent.com/22185279/151248044-17c46389-572c-42d0-93f2-057b2fd6c35a.png)